### PR TITLE
 #2203 issue fix

### PR DESCRIPTION
**Pull request by mhmdzaid**
## Description
 #2203 issue fix ,  onBoardingViewController second next scene pushed only once now 
## Screenshots
ezgif-2-d789310e73eb

---
Internal Tracking ID: [EXPOSUREAPP-5771](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5771)

### CODE REVIEW

**Comment by mozead1996** [comment]
@nickguendling  hi nick this is my first PR to open source project why  it's keep telling me
  build-and-test Expected — Waiting for status to be reported ?

**Comment by Ein-Tim** [comment]
@mozead1996 Can you update the PR so that it matches the current HEAD ("Update Branch"), maybe the check will run then.

**Comment by Ein-Tim** [comment]
Weird afaik the checks should run instantly...
[Circleci](https://status.circleci.com/) has a minor service outage, but macOS shouldn't be affected...

File: src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift [hunk 1]
```diff
@@ -394,7 +394,9 @@ final class OnboardingInfoViewController: UIViewController {
 			persistTimestamp(completion: completion)
 		case .enableLoggingOfContactsPage:
 			func handleBluetooth(completion: @escaping () -> Void) {
-				if let alertController = self.exposureManager.alertForBluetoothOff(completion: { completion() }) {
+				if let alertController = self.exposureManager.alertForBluetoothOff(completion: {(
+					
+				)}) {
 					self.present(alertController, animated: true)
 				}
 				completion()
```
Comment (KaiTeuber) [inline src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift:402]: I'm not quite sure, but as far as I can test, without a device at the moment, the call inside the closure seems to be correct. It's a special case in Debug build settings.
I guess removing line 400 402 should solve the problem.

File: src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.xib [hunk 2]
```diff
@@ -249,6 +249,32 @@
...
+        <designable name="vh2-Dv-3W6">
+            <size key="intrinsicContentSize" width="33" height="16"/>
+        </designable>
+        <designable name="vo8-Zn-enn">
+            <size key="intrinsicContentSize" width="54" height="50"/>
+        </designable>
+        <designable name="znr-XX-rfp">
+            <size key="intrinsicContentSize" width="85" height="18"/>
+        </designable>
+    </designables>
```
Comment (KaiTeuber) [inline src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.xib:277]: Why are these changed as well, it is required for the fix?

**Comment by KaiTeuber** [comment]
Thanx for the support - we will review the bugfix

**Review by KaiTeuber — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.xib (hunk 2 above)
Comment (mozead1996) [reply src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.xib:277]: thanks for your reply ! no these changed by when I opened xib file of OnboardingViewController.xib , not required for the fix @KaiTeuber

File: src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift (hunk 1 above)
Comment (mozead1996) [reply src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift:402]: but if I remove 400 , 402 , what is usage of handleBluetooth(completion:) ?
@KaiTeuber

**Comment by dsarkar** [comment]
---
Internal Tracking ID: [EXPOSUREAPP-5771](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5771)

**Review by mhmdzaid — COMMENTED** [review]
@KaiTeuber  here I made the changes

File: src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift [hunk 3]
```diff
@@ -395,9 +395,7 @@ final class OnboardingInfoViewController: UIViewController {
 		case .enableLoggingOfContactsPage:
 			func handleBluetooth(completion: @escaping () -> Void) {
 				if let alertController = self.exposureManager.alertForBluetoothOff(completion: { completion() }) {
-					self.present(alertController, animated: true)
```
Comment (whiskey) [inline src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift:398]: Don't you think the `handleBluetooth` function is redundant then after this change? So why is this not removed at all and and removed from all the subsequent calls, like in `askExposureNotificationsPermissions` below?

**Review by whiskey — CHANGES_REQUESTED** [review]
